modules/UI/side_pannels/profile/Profile.js

```diff
@@ -1,10 +1,45 @@
-/* global $ */
+/* global $, _ */
 import UIUtil from "../../util/UIUtil";
 import UIEvents from "../../../../service/UI/UIEvents";
 import Settings from '../../../settings/Settings';
 
+const sidePannelsContainerId = 'sideToolbarContainer';
+const compiledTpl = _.template(`
+    <div id="profile_container" class="sideToolbarContainer__inner">
+        <div class="title" data-i18n="profile.title"></div>
+        <div class="sideToolbarBlock first">
+            <label class="first" data-i18n="profile.setDisplayNameLabel">
+            </label>
+            <input type="text" id="setDisplayName"
+                data-i18n="[placeholder]settings.name" 
+                placeholder="Name">
+        </div>
+        <div class="sideToolbarBlock">
+            <label data-i18n="profile.setEmailLabel"></label>
+            <input type="text" id="setEmail" placeholder="Enter e-mail">
```

Comment:
This is the current HTML for this input:
`<input data-i18n="[placeholder]profile.setEmailInput" type="text" id="setEmail">`